cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java

```diff
@@ -131,7 +131,6 @@ public void putSecureData(String namespace, String name, byte[] data, String des
       keyStore.setKeyEntry(keyName, new KeyStoreEntry(secureStoreData, meta), password, null);
       // Attempt to persist the store.
       flush();
-      LOG.info(String.format("Successfully stored %s in namespace %s", name, namespace));
```

Comment:
Hmm.. yeah I will make it debug. Seems like good info to have while debugging.


> On Aug 2, 2016, at 5:08 PM, Bhooshan Mogal <notifications@github.com> wrote:
> 
> In cdap-security/src/main/java/co/cask/cdap/security/store/FileSecureStore.java <https://github.com/caskdata/cdap/pull/6274#discussion_r73261135>:
> 
> > @@ -131,7 +131,6 @@ public void putSecureData(String namespace, String name, byte[] data, String des
> >        keyStore.setKeyEntry(keyName, new KeyStoreEntry(secureStoreData, meta), password, null);
> >        // Attempt to persist the store.
> >        flush();
> > -      LOG.info(String.format("Successfully stored %s in namespace %s", name, namespace));
> I meant you could just reduce this to debug, but your call
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/caskdata/cdap/pull/6274/files/b5c602cb63a60e61be6506a3072b89a5be3e40aa..15f0f9e182f2efce31a1667082ccb7a1296e1b8c#r73261135>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAFGk4LQ7pPm1RHSOkqTWYaDT8RrFQSpks5qb9wBgaJpZM4JZI-3>.
>